src/Native/CMakeLists.txt

```diff
@@ -26,10 +26,19 @@ endif ()
 
 if (CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64)
     add_definitions(-DBIT64=1)
-    add_definitions(-DX64=1)
+    add_definitions(-D_AMD64_)
+elseif (CMAKE_SYSTEM_PROCESSOR STREQUAL x86)
```

Comment:
Is this just fixing x86? I see that we are already checking for it down in pal_runtimeinformation.cpp, but I don't actually see it getting defined anywhere without this change